controllers/config/config.go

```diff
@@ -56,10 +60,6 @@ func fillConfig(clusterConfig *configv1.Network, operConfig *operatorv1.AntreaIn
 	if !ok {
 		antreaAgentConfig[types.DefaultMTUOption] = types.DefaultMTU
 	}
-	// Set Antrea image.
-	if operConfig.Spec.AntreaImage == "" {
-		operConfig.Spec.AntreaImage = types.DefaultAntreaImage
-	}
```

Comment:
Is this change unrelated to this PR?

Reply:
Moved to line 146 within fillConfig() as it's not agent nor controller param